packages/plugins/ansible-collection-theforeman-foreman/ansible-collection-theforeman-foreman.spec

```diff
@@ -20,10 +20,8 @@ Provides: ansible-collection(%{collection_namespace}.%{collection_name}) = %{ver
 Provides: bundled(python-apypie) = 0.5.0
 
 Requires: ansible-core
-Requires: (python3-requests if ansible-core >= 1:2.14.7)
-Requires: (python3-pyyaml if ansible-core >= 1:2.14.7)
-Requires: (python3.11-requests if (ansible-core >= 2.14.2-3 and ansible-core < 1:2.14.7))
-Requires: (python3.11-pyyaml if (ansible-core >= 2.14.2-3 and ansible-core < 1:2.14.7))
+Requires: (python3.12-requests if (ansible-core >= 2.14.2-3 and ansible-core < 1:2.17))
+Requires: (python3.12-pyyaml if (ansible-core >= 2.14.2-3 and ansible-core < 1:2.17))
```

Comment:
No, you want `python3.12-*` only for `1:2.16.14-3` and newer. So, how about:
```
Requires: (python3-requests if (ansible-core >= 1:2.14.7 and ansible-core < 1:2.16.14-3))
Requires: (python3-pyyaml if ((ansible-core >= 1:2.14.7 and ansible-core < 1:2.16.14-3))
Requires: (python3.11-requests if (ansible-core >= 2.14.2-3 and ansible-core < 1:2.14.7))
Requires: (python3.11-pyyaml if (ansible-core >= 2.14.2-3 and ansible-core < 1:2.14.7))
Requires: (python3.12-requests if ansible-core >= 1:2.16.14-3)
Requires: (python3.12-pyyaml if ansible-core >= 1:2.16.14-3)
```

Reply:
Sometimes I forget how to do packaging.